### PR TITLE
run cost_explorer policy for haim-account

### PR DESCRIPTION
**Pull request by athiruma**

### CODE REVIEW

File: jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 # os.system(f"""podman run --rm --name cloud-governance -e AWS_DEFAULT_REGION="us-east-1" -e account="psap" -e policy="cost_explorer" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PSAP}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PSAP}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e es_index="{es_index_global}" -e cost_explorer_tags="{cost_tags}" -e granularity="{granularity}" -e cost_metric="{cost_metric}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
 # os.system(f"""podman run --rm --name cloud-governance -e AWS_DEFAULT_REGION="us-east-1" -e account="perf-scale" -e policy="cost_explorer" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e es_index="{es_index_global}" -e cost_explorer_tags="{cost_tags}" -e granularity="{granularity}" -e cost_metric="{cost_metric}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
 
-es_index_global = 'cloud-governance-cost-explorer-global-cost'
+es_index_global = 'cloud-governance-cost-explorer-perf-global-cost'
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:47]: Why perf, is it for Haim only ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:47]: No, a small update to our accounts

Comment (ebattat) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:47]: Try not to mix next time

**Review by ebattat — COMMENTED** [review]
Did you add the grafana dashboad for all the policies and cost by daily/user/manager as we have for our account

**Comment by athiruma** [comment]
> @athiruma, Did you add the Grafana dashboard for all the policies and cost by daily/user/manager as we have for our accounts ?

I will after this pr merge

**Pull request merged**